Microsoft.DotNet.Interactive/LanguageService/Range.cs

```diff
@@ -5,7 +5,13 @@ namespace Microsoft.DotNet.Interactive.LanguageService
 {
     public class Range
```

Comment:
Any reason not to use `TextSpan`? The new parsing code uses this and a few other types from `Microsoft.CodeAnalysis.Text`.